src/coreclr/jit/emit.cpp

```diff
@@ -1020,23 +1020,28 @@ insGroup* emitter::emitSavIG(bool emitAdd)
         }
     }
 
-    // Fix the last instruction field
+    // Fix the last instruction field, if set. Note that even if there are instructions in the IG,
+    // emitLastIns might not be set if an optimization has just deleted it, and the new instruction
```

Comment:
I believe this is future proofing the "delete last instruction" feature, right?

Reply:
Yes, I added this to the "delete last instruction" PR, so this is just extracting that part and submitting it early.